Tic-Tac-Toe RSpec test

GzimAsani opened the pull request:
Millstones:
- We first tested all the instance methods on Player Class
- Then we tested all instance methods on Game Class
- Avoided methods that are dependent on other methods

GzimAsani committed:
Added rspec for player class

tahiry-dev committed:
start the basic test of game.rb

tahiry-dev committed:
test the game class

tahiry-dev committed:
fix linters

andresporras3423 reviewed (changes requested):
### Status: Changes Required :recycle:
**Hello Tahiry Randriamiarintsoa & Gzim Asani** 
Good job with your first submission :muscle:  :muscle:
Your project meets most of the requirements :clap: :clap: :clap:  
There are just a couple of details to improve before you can go ahead:

- [x] Fix rubocop offenses
- [x] Test negative scenarios, you might use "to_not eql" and "to_not be_an"
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

GzimAsani committed:
Fixed rubocop offenses

GzimAsani committed:
Add negative scenarios

GzimAsani commented:
> ### Status: Changes Required 
> **Hello Tahiry Randriamiarintsoa & Gzim Asani**
> Good job with your first submission  
> Your project meets most of the requirements   
> There are just a couple of details to improve before you can go ahead:
> 
> * [ ]  Fix rubocop offenses
> * [ ]  Test negative scenarios, you might use "to_not eql" and "to_not be_an"
> 
> Please request another review once you are ready! 
> 
> Happy coding 
> 
> Oscar Andrés Russi Porras

Thank you for your review. We have fixed the rubocop offenses and added the negative scenarios.

voscarmv reviewed (approved):
# Project status: Approved ✨ 😎👍 💯 ✔️
### You may merge this branch and move forward ⛙🛣

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️
